CommunityGuidelines.md

```diff
@@ -0,0 +1,61 @@
+# Community Guidelines
+
+## Community rules
+
+- Please be respectful to other members. Avoid any racist, sexist, homophobic, excessively political, or otherwise distasteful discussions.
+- While some cursing is appropriate in context, refrain from using excessive profanity.
+- Keep conversations in the appropriate channel. Some off-topic discussions are to be expected but try not to blatantly post non-subject-related content.
+- No NSFW material is allowed. Breaking this rule will be met with an immediate ban.
+- Keep spam to a minimum:
+  - Do not repeatedly send the same message.
+  - Do not post links to sites/content others would find unacceptable.
+- Promoting your content, creations, YouTube channel, etc. is encouraged, but you must have permission from staff to advertise another Discord community.
+- Advertising or promotion of piracy is forbidden.
+- Staff members (`@Developer` on Discord) have the final say about what is and is not acceptable within the community. Refusals to listen to staff may result in being kicked or banned.
+- HAVE FUN and enjoy all that the community has to offer.
+
+To read our Code of Conduct please refer to `CODE_OF_CONDUCT.md`.
+ 
+## Team rules
+
+On top of the community rules the development and moderation team has to keep themselves to a specific set of rules.
+
+### Behavioural rules
+
+- Keep cool to any community member, even if they can be very frustrating to deal with.
+- Stand up for one another and take responsibility for each others mistakes.
+- Any team member can request a time-out in team chat, which will suspend any talking in team chat for 30 minutes.
+  - Respect the time out, do not contact each other in DM during this period.
+- We work with a 3 strike system, if you hit strike 3 you will lose your role.
+
+### Rules regarding PRs
+- Any contributor PR requires at least two code reviews, either by developers or trusted community members (at least one developer).
+- Any developer PR requires at least one code review, by another developer.
+- Any PR requires one playtest review, if applicable.
+
+### Guidelines regarding public relations and community management
+
+With an online presence we require maintaining a good public image. For this we have specific guidelines, so each party can take responsibility where necessary.
+
+- Any message* for support or public relations requires a signature**.
+- When signing an email please use the template
```

Comment:
```suggestion
- When signing an email please use the template.
```

Reply:
This line should be edited in general, beside punctuation, because we decided to remove this template.

Perhaps "Close any email off with sending your regards, your name, and the OpenRCT2 Team."

Reply:
Applied above mentioned.